src/qt/walletmodel.cpp

```diff
@@ -203,7 +203,7 @@ WalletModel::SendCoinsReturn WalletModel::prepareTransaction(WalletModelTransact
         int nChangePosRet = -1;
```

Comment:
In 7beb338a0d4343a622236876c7d63d56bf7039e3 "wallet: avoid createTransaction() with signer"

Nit in commit message: 
```diff
- wallet: avoid createTransaction() with signer
+ wallet: avoid signing via createTransaction() with external signer
```